packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -29,6 +29,8 @@ import { PlaySubscriptions } from '../payments/google-play/subscriptions';
 import { PayPalHelper } from '../payments/paypal/helper';
 import { StripeHelper } from '../payments/stripe';
 import { AuthLogger, AuthRequest } from '../types';
+import doc from './utils/docs/account_docs';
+import D from './utils/docs/descriptions';
```

Comment:
Personally, I'm not a fan of single-letter names like this and think `DESCRIPTIONS` would be more clear when using it, but it's a personal preference. :)